BoxSDK.xcodeproj/xcshareddata/xcschemes/TestApp.xcscheme

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Just making sure: you want to commit this file?  Is it for the Test App you created recently or for the existing Test App in the project?

Reply:
This file is redundant. I've already removed it in the next commit.